src/pages/member/EventCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import withStyles from '@material-ui/core/styles/withStyles'
+import Typography from '@material-ui/core/Typography'
+import Card from '@material-ui/core/Card'
+import { COLOR } from '../../constants/Constants'
+
+const styles = {
+    page: {
+        width: '345px',
+        marginRight: '34px',
+        marginTop: '27px',
+    },
+    title: {
+        color: COLOR.TITLE_GREEN,
+        fontWeight: 'bold',
+        fontSize: '36px',
+        paddingLeft: '37px',
+        paddingTop: '10px',
+        paddingBottom: '15px'
+    },
+    eventName: {
+        fontSize: '23px',
+        paddingLeft: '37px',
+        fontWeight: 'bold'
+    },
+    eventDate: {
+        fontSize: '20px',
```

Comment:
should this be 24?

Reply:
![image](https://user-images.githubusercontent.com/45786074/85912526-ebcfa500-b7e1-11ea-9074-33e577cfcf68.png)
imo looks better with eventDate slightly smaller - thoughts?

Reply:
tlaked to jasmine, figma is updated with new design